fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -784,14 +784,17 @@ StringRef StringRefOf(const char* s) {
   return StringRef((uint8_t*)s, strlen(s));
 }
 
-void SimulationConfig::generateNormalConfig(int minimumReplication, int minimumRegions) {
+void SimulationConfig::generateNormalConfig(int minimumReplication, int minimumRegions, int fearlessOverride) {
```

Comment:
I think this would make more sense as a `maximumRegions == 0`

Reply:
again this commit is removed.